### DISCUSSION TITLE
Any way to turn off the air raid siren?

### DISCUSSION THREAD

**Discussion by coffeecoffeecoffee** [TEHE-Studios/ExpandedHelicopterEvents#257] [Q&A]
Hi! Is there any way to just disable the air raid siren? I'm playing the game with a friend who has some trauma regarding air raids, and we absolutely adore everything else in the mod, but the constant sirens for days is severely upsetting for her.

**Comment by Chuckleberry-Finn** [comment]
The air raid shouldn't be lasting for days, it's been an on-going issue with syncing. She should be able to reconnect and the siren should go away. Being this is also an issue with trauma, I can look into adding a 'disable entirely' feature.

**Comment by coffeecoffeecoffee** [comment]
Hey, it's okay! Especially if it lasts less than a day, you don't by any means need to add a feature just for potentially one person. Thank you so much, reconnecting made it go away for me, too, so that seems to be what happened here.

**Reply by frigoulash** [reply]
I've had the blaring siren as well. Had to do a server restart to get it to end. A command into the server console could be a great alternative as well imo.

**Comment by VRoweUA** [comment]
I'm Ukrainian and air raid sirens also cause me discomfort, I hear them nearly daily, don't want to hear them ingame and wonder if it's real or fake
+1 on this issue, It would be amazing if you could disable the sirens altogether, thanks!

**Reply by sharkster91** [reply]
We will go ahead and create a disable feature for air raid sirens since you
already have enough trouble with cruise missiles that you don't need them
in Zomboid.

**Comment by Chuckleberry-Finn** [comment]
I can move this up the to-do.
